### PR TITLE
Added images for classic fanatics

### LINKED ISSUE #2697: Classic Fanatics generic art for active buttons
Thanks to two nights of insomnia, I have the active-without-artwork Fanatics done up with the help of Fog's legacy assets.  Thanks to @blackshadowshade for allowing me the honor of using Fog's files.

Now these need to be looked over with a fine tooth'd comb to make sure the dice match up, the names don't bleed over the edge or into the frames/filligree, and any other things that stick out as plain wrong.

Already noticed a few minor errors, but fixed all but one.

As I said, these do need to be checked.  I'll start to work on the Inactive ones at some point.

barswanian
bobby5150
Coyote
Cristofore
Darthcliff
Discordia
fxdirect
GorgorBey
grayhawk
GreatWolf
Grivan
Hrodgar
IIconfused
Jordan
kestrel
maga
nelde
NoopMan
Notorious
Pain
Raupe
SailorMur
santiago
SC(TheDeuce)
Snuff
spindisc
Squiddhartha
Stick
TheMachine
Torch
Totoro
trifecta
TYFTFB
wtrollkin
zaph
Zomulgustar

### PR DESCRIPTION
**Pull request by blackshadowshade**
Fixes #2697.

I'll post screenshots of what I'm seeing on my Classic Fanatics set page, that'll help somewhat with verifying recipes, I hope.

### CODE REVIEW

**Comment by blackshadowshade** [comment]
Here are the screenshots of all the Classic Fanatics:
Screen Shot 2021-03-27 at 2 00 59 pm
Screen Shot 2021-03-27 at 2 02 03 pm
Screen Shot 2021-03-27 at 2 02 14 pm
Screen Shot 2021-03-27 at 2 02 28 pm
Screen Shot 2021-03-27 at 2 02 36 pm
Screen Shot 2021-03-27 at 2 02 44 pm
Screen Shot 2021-03-27 at 2 02 55 pm
Screen Shot 2021-03-27 at 2 03 03 pm
Screen Shot 2021-03-27 at 2 03 13 pm
Screen Shot 2021-03-27 at 2 03 27 pm
Screen Shot 2021-03-27 at 2 03 35 pm
Screen Shot 2021-03-27 at 2 03 45 pm
Screen Shot 2021-03-27 at 2 03 52 pm
Screen Shot 2021-03-27 at 2 04 01 pm
Screen Shot 2021-03-27 at 2 04 07 pm

**Comment by blackshadowshade** [comment]
I note that Darrin's recipe doesn't seem to match the image.

**Comment by blackshadowshade** [comment]
Apart from that, all the other new images look good to me.

**Comment by TheOrgg** [comment]
I matched Binder's artwork to the recipe.
150pxBinder

Darrin's recipe is indeed off; that was using Fog's original file.  It's off in the Classic Fanatics set right now.  I guess nobody noticed until now.  Like all but one (RavenBlack, I think?) the recipe was correctly labeled, just with the wrong value typed into the text field.  This one makes me feel even stronger towards changing RavenBlack to match the artwork.  Who had the "master list" of Fanatics and/or input the recipes? 
150pxDarrenFanatic

Blubec's recipe is out of order, swapping the stinger and ornery's positions would match the art.


Noticed an inconsistency with Coyote.  Fog used the fully divided Dice Frame to show unskilled Option dice, but I used the fraction.  Do I need to correct this?  I'll list any others I spot like that here.

Dwelsh is an EBEL Haxor.  Just got that first part. lol.

Gorgorbey's warrior die feels weird in the middle of the recipe, but that's a 'whatever,' really.

I've done through Kirosaki, but now I need breakfast.  I'll make it a quick sweet pickle and cheese sandwich and be back in a few.

**Comment by TheOrgg** [comment]
Export got hung up for Darrin, here's the correct 150px.

150pxDarrenFanatic

Lady J's recipe is all out of order compared to the art.

Found an error on this guy, the Chance formatting was messed up.
150pxNihlathak

Prof. Booty's recipe is out of order compared to the image.

There's RavenBlack again.  

This doesn't matter right now, but Tasselhoff B's recipe has speed in both choices of Plasma, but probably should have speed as an ability then the two plasma options.  No clue if that's important or not, but to match it with the Space Girl would feel correct.

UrbanMyth's d15 is a z(15) on the artwork.  This is another "RavenBlack" case, too.  Fog has it labelled as a Speed in the .psp file.  Usually when there's an error like this, the layer is labelled matching the recipe, but the text in the text field is wrong.  Should I make another pull request for RavenBlack and UrbanMyth?

Zomulgustar's Z? should have a purple fill, but that's minor-- especially with just how many skills are on that die.  That's one I'd have to work a while on, so I'm leaving it unless anyone strongly objects.

**Comment by blackshadowshade** [comment]
I've been thinking about the mismatch between the recipes we have on the site and the graphics that Fog created.

I think we should lean towards making the image match the recipe unless we have a really good reason to think that the recipe on the site is wrong, since the recipes we have for the Classic Fanatics come directly from Chaos's data scrape from the old site, so we have reason to believe that they are correct (except for g/G mixups like gman).

**Comment by blackshadowshade** [comment]
As a follow-up, if we're changing the image to match the recipe, then I'm happy to do that in this pull request, no need to open another issue.

**Comment by blackshadowshade** [comment]
I've updated the files for Darrin and nihlathak.

**Comment by TheOrgg** [comment]
Ok.  If it had been Fog who kept the list and put them on the site, the argument for changing the recipe held more merit.  Since it wasn't, I concede the issue.  My beliefs on why they might be correct were based on his psp files.

150pxRavenBlackh
150pxUrban

**Comment by blackshadowshade** [comment]
I've updated the images for RavenBlack and Urbanmyth.

**Comment by TheOrgg** [comment]
Where's the site that's getting the images on it?  I'd like to go through it and look at the updated bits.

**Comment by blackshadowshade** [comment]
There is no official dev site for this, I produced those screenshots from the local dev site on my computer.

@cgolubi1, a dev site for this could be really useful.

**Comment by TheOrgg** [comment]
Could you put it on Staging since it's easy to revert?  That's what I had assumed, honestly.

That'd allow me and possibly @dwvanstone to comb through the set and make sure the whole issue is fixed.

**Comment by blackshadowshade** [comment]
Unfortunately, no pull request is truly easy to revert, which is why we do individual test sites before merging them into the code base.

**Comment by cgolubi1** [comment]
These images should be up at http://momentary.dev.buttonweavers.com/ui/ --- check them out.

**Comment by TheOrgg** [comment]
Binder's still the incorrect image.

Found an error on roujin; had a n(8) instead of the p(8) listed in the recipe.

Zog's reserve Time and Space is functionally just a M(2).   I lol'd.

**Comment by TheOrgg** [comment]
Forgot the images.  D'oh!

Jeff's Ornery was uncolored.  Fixed that.  Also applied the "shine" to these two.

150pxJeffBoy
150pxroujin

**Comment by blackshadowshade** [comment]
I've updated the images for jeffboyardy and roujin27, and am awaiting the others.

**Comment by TheOrgg** [comment]
It doesn't look like they were updated, nor Binder.  I also went ahead and fixed Coyote and Sailor Mur's minor Option Dice issue.
150pxcoyote
150pxSailorMur

Is the "shine" a big enough issue to reexport all the previously recreated images?  I have all of them, so I could re-export and make the shine/shade consistent throughout the set.

**Comment by blackshadowshade** [comment]
I'm seeing the updated buttons on momentary.

However, @TheOrgg, I never had a new image for Binder, I don't know where you posted that.

As for shine/shade, I'm happy to update the images if you provide them.

**Comment by blackshadowshade** [comment]
Oh, let me rephrase. I've updated the two buttons in the pull request. The dev site has not been updated yet, that's for Chaos to do, and she will comment explicitly when she does that.

**Comment by TheOrgg** [comment]
Oh, okay.  I get how that works, now.  You can't update Momentary.bw.com, only GlassOnion.  I didn't know that.

Binder is up there with the incorrect z(4) recipe Darrin on this thread.  It's subtle-- one number in the option die.  Like the previous two, it's hard to notice.

**Comment by blackshadowshade** [comment]
Thanks for pointing me to the Binder image. I've updated the pull request with the new Coyote, SailorMur, and Binder images.

**Comment by blackshadowshade** [comment]
I note that the Binder image has no shadow or shine.

If you can do the shine/shadow, that would be great to maintain consistency, but I'm not sure how much work that is for you.

**Comment by TheOrgg** [comment]
It's about 160 images, but it only takes about six or seven minutes each because they're already fixed up for the most part.

**Comment by blackshadowshade** [comment]
In that case, prioritise those that have no shadow, I would say. If you have the time and inclination after that, go back and add shine to those with no shine.

**Comment by blackshadowshade** [comment]
On cursory inspection, the buttons without any shadow effect are: Binder, Nihlathak, RavenBlack, and UrbanMyth.

**Comment by TheOrgg** [comment]
This was a bit more detail than expected; the computer's slow with that many images in memory.  That said, I'm through the F's.  I'll pick up again when I have time, probably tomorrow.  I'm heading to bed now.

**Comment by TheOrgg** [comment]
150pxalbertel
150pxAllJazzedup
150pxAnders
150pxAnthony
150pxAnthrem
150pxAnti-Llama
150pxAysez
150pxBarswanian
150pxbigevildan
150pxBinder
150pxBirdman
150pxBlackOmega
150pxBobby5150
150pxBodie
150pxBonefish
150pxboot2dahead
150pxBowler
150pxBraincraft
150pxBrashTech
150pxBugRoger
150pxCactusJack
150pxCaligari
150pxCalmon
150pxChase
150pxConspyre
150pxCoyote
150pxCristofore
150pxDarthcliff
150pxDiscordia
150pxDocBlue
150pxDownen
150pxDurak
150pxDwelsh
150pxEchopapa
150pxEon
150pxFamous
150pxFendren
150pxFernworthy
150pxFinngall
150pxFlesh99
150pxFnord
150pxFxDirect
150pxGhostWalker
150pxGoldenOtter
150pxGorgorBey
150pxGrayhawk
150pxGreatWolf
150pxGrend
150pxGripTiger
150pxGrivan
150pxGrym
150pxGryphon
150pxgslamm
150pxhairlesswonder
150pxHaruspex
150pxHooloovoo
150pxHrogdar
150pxIcarus
150pxifni
150pxiiConfused
150pxInnundator
150pxJasyeman
150pxJeffBoyardy
150pxJennie
150pxJGenzano
150pxJordan
150pxJrBrown
150pxJuelki
150pxKaeriol
150pxKaijuGamer
150pxKaufman
150pxKleric
150pxKurosuke
150pxLacky
150pxLimax
150pxLlippen
150pxLukeii
150pxLunatic
150pxMaga
150pxMalarson
150pxMLvanbie
150pxMneme
150pxMoekon
150pxMoodster
150pxmoussambani
150pxMushu
150pxMyxozoa
150pxnelde
150pxNeoVid
150pxNihlathak
150pxNoopMan
150pxNotorious
150pxPain
150pxPGolo
150pxPjack
150pxRandomLife
150pxRaupe
150pxRavenBlack
150pxRoujin27

**Comment by TheOrgg** [comment]
Note:  It looks like there's a 100 image limit to posts.  It also wouldn't let me post these without refreshing the page.
150pxRumbles
150pxSabathia
150pxSailorMur
150pxSanny
150pxSantiago
150pxSC(TheDeuce)
150pxSchwa
150pxSeeker
150pxShadowKeeper
150pxSlamKrypare
150pxSnuff
150pxSpinDisc
150pxStick
150pxSyberCat
150pxTasselhoff
150pxTheFool
150pxTheMachine
150pxTorch
150pxTotoro
150pxTrifecta
150pxTrouble
150pxTweedle
150pxTYFTFB
150pxUncleMilo
150pxUrbanMyth
150pxVhooDoo
150pxWembleyFraggle
150pxWeylan
150pxWranklePig
150pxWTrollkin
150pxYagHarek
150pxZaph
150pxZomulgustar
150pxZotmeister

**Comment by blackshadowshade** [comment]
All images updated.

@cgolubi1, could we get an update to the dev site please?

**Comment by TheOrgg** [comment]
Reposting the link down here so it's easier to find.  http://momentary.dev.buttonweavers.com/ui/

I'm going to make a post on the forum for anyone to check over this, too.  At this point, my eyes are nearly glazed over looking at the set.  lol.

**Comment by dwvanstone** [comment]
So much niceness here! I love the cleaner shadow look on the buttons. Yes I see the newly yellowed ornery dice. I like the new haruspex! The new kestrel is beautiful.

I only focused on the buttons which will show up on the live site. (I figure if ever we get the others onto the site we can review them then.)

fxdirect has a p(X,X)! but the image only has a single "!". (Compare to CestWhat which has two "!"s which is easier to read.)

That's the only comment I have. It all looks so good!!

**Comment by dwvanstone** [comment]
Okay - then CestWhat should be consistent and have two T's with a single exclamation point between them.

**Comment by TheOrgg** [comment]
I don't have the art elements for CestWhat.  That's not one of the new-art ones.  You can tell because of the font.  That was submitted by the user CestWhat at some point before I came to buttonweavers.

Had there been a blank for it (like the Rage Brothers or Lil' Sven, I could try, but there isn't that I have found)

**Comment by dwvanstone** [comment]
Okay - sounds good to me then.

**Comment by TheOrgg** [comment]
@blackshadowshade , unless you spot an issue that needs fixing in the images, I think we can go ahead and slot this in for the next batch of updates.

**Comment by blackshadowshade** [comment]
Sounds good to me. We can do subsequent tweaks and corrections when they're spotted in other pull requests.

@cgolubi1, this is now ready for you to review and accept.

**Comment by blackshadowshade** [comment]
Argh. @cgolubi1, could you take a look at this CircleCI failure and see why it's failing? If I read it correctly, JSHint isn't installing correctly?

**Comment by cgolubi1** [comment]
Will do, but given "this is an error with npm itself" and the fact that we haven't changed anything, i think we should try it again in a day to see if it's a more general transient problem that the repo maintainers fix for us sooner than we could do anything.

**Comment by blackshadowshade** [comment]
I note a number of "deprecated" lines:

    npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
    npm WARN deprecated coffee-script@1.3.3: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)

    npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
    npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js

    npm WARN deprecated nodeunit@0.7.4: you are strongly encouraged to use other testing options

Maybe these have caught up with us.  :)

I'll try triggering the unit tests tomorrow.

**Comment by blackshadowshade** [comment]
I've run the unit tests two more times and they're still failing at the same juncture.

**Comment by blackshadowshade** [comment]
I've rebased this pull request over the npm change.

**Comment by cgolubi1** [comment]
Merging!  Thanks, TheOrgg!
